src/lib/mina_base/test/dune

```diff
@@ -1,20 +1,29 @@
 (tests
- (names zkapp_account_test)
+ (names mina_base_test)
```

Comment:
I think it would be nice to have a naming discipline that helps in easily differentiating the "main" test file from all others, without having to look at the dune file. Here, the name is quite similar to all the other test files which does not help. What do you think of calling it simply `main` or `mina_base_tests_main` ?